### DISCUSSION TITLE
How to clean all temporary files after install or upgrade

### DISCUSSION THREAD

**Discussion by xbotuk** [microsoft/vcpkg#42348] [Q&A]
How to clean all temporary files after install or upgrade

Can this be defaulted through a config or system param?

**Comment by autoantwort** [comment]
Yes you can pass `--clean-after-build `

**Reply by Osyotr** [reply]
I prefer `--clean-buildtrees-after-build --clean-packages-after-build` because `--clean-after-build` implies `--clean-downloads-after-build` which results in many msys2 redownloads on windows.

https://learn.microsoft.com/en-us/vcpkg/commands/install#clean-after-build
